components/Banner/src/MDCBannerView.m

```diff
@@ -352,15 +353,15 @@ - (CGSize)sizeThatFits:(CGSize)size {
     }
     case MDCBannerViewLayoutStyleMultiRowAlignedButton: {
       frameHeight += kTopPaddingLarge + kBottomPadding;
-      frameHeight += [self getFrameHeightOfImageViewAndTextLabelWithSizeToFit:size];
+      frameHeight += [self getFrameHeightOfImageViewAndTextLabelWithSizeToFit:adjustedSizeToFit];
```

Comment:
Tip: Should this change for multi-row aligned button layout result in a visible difference? If not, please revert this change. If it should be visible, please add a test for it.

Reply:
This is aligned style but only single button. This one will be kept in this PR for snapshot comparison. I added a separate snapshot for stacked style in 68abaf82d01c639d7a0403f674f36db4ce5f5ed6.